chanbackup/pubsub.go

```diff
@@ -97,14 +97,25 @@ type SubSwapper struct {
 
 	quit chan struct{}
 	wg   sync.WaitGroup
+
+	// includeCloseTxInputs specifies if to put CloseTxInputs into a backup.
```

Comment:
nit: "specifies if we should put the CloseTxInputs into..."?

Reply:
Fixed. Also, fixed another instance in `chanbackup/backup.go`.